src/EditorFeatures/Test2/Classification/ClassificationTests.vb

```diff
@@ -114,24 +114,23 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Classification
         End Function
 
 #Disable Warning BC40000 ' Type or member is obsolete
```

Comment:
:question: Is this still needed, or have we removed references to obsolete code at this point?